src/main/features/response/helpers/taskListBuilder.ts

```diff
@@ -137,6 +133,37 @@ export class TaskListBuilder {
       )
     }
 
+    if (partiallyAdmitted && HowMuchDoYouOweTask.isCompleted(draft)) {
```

Comment:
Do not duplicate `partiallyAdmitted` condition, use previous block started line 106 (with `if (partiallyAdmitted) {`)

Reply:
done